src/components/Person/Person.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+import { PersonType } from '../../types';
+
+type Props = {
+  person: PersonType;
+  selectedPersonSlug: string;
+  isMother: PersonType | null;
```

Comment:
The `isFather` prop should have a more appropriate name like `father` since it represents a `PersonType` or null.